### PR TITLE
add about details

### PR DESCRIPTION
**Pull request by auryn-macmillan**
Added some about details and a link to the blog and forum.

### CODE REVIEW

File: vue-app/src/App.vue [hunk 1]
```diff
@@ -62,7 +64,7 @@ html {
 }
 
 a {
-  color: $text-color;
+  color: $highlight-color;
```
Comment (xuhcc) [inline vue-app/src/App.vue:67]: Please don't change the base stylesheet.

File: vue-app/src/views/About.vue [hunk 2]
```diff
@@ -1,15 +1,37 @@
 <template>
   <div class="about">
-    <h1 class="content-heading">About</h1>
-    <p>Clr.fund is a protocol for efficiently allocating funds to public goods that benefit the Ethereum Network according to the preferences of the Ethereum Community. Clr.fund strives for credible neutrality, decentralization, permissionlessness, trustlessness, and pseudonymity.</p>
+    <h1 class="content-heading title">About</h1>
+    <p class="content">Clr.fund is a protocol for efficiently allocating funds to public goods that benefit the Ethereum Network according to the preferences of the Ethereum Community. Clr.fund strives for credible neutrality, decentralization, permissionlessness, trustlessness, and pseudonymity.</p>
+
+    <h2 class="content-heading">Why does clr.fund exist?</h2>
```
Comment (xuhcc) [inline vue-app/src/views/About.vue:6]: `.content-heading` is the class for top-level heading inside `#content` block. It should not apply to other elements. It is also redundant.

**Review by xuhcc — CHANGES_REQUESTED** [review]
I think this page should be styled similarly to project page. I can do that if you want

File: vue-app/src/views/About.vue (hunk 2 above)
Comment (auryn-macmillan) [reply vue-app/src/views/About.vue:6]: > I think this page should be styled similarly to project page. I can do that if you want

Yeah, that would be great!

**Pull request merged**